openmdao/core/problem.py

```diff
@@ -96,11 +96,14 @@ class Problem(object):
     _initial_condition_cache : dict
         Any initial conditions that are set at the problem level via setitem are cached here
         until they can be processed.
-    _setup_status : int
-        Current status of the setup in _model.
-        0 -- Newly initialized problem or newly added model.
-        1 -- The `setup` method has been called, but vectors not initialized.
-        2 -- The `final_setup` has been run, everything ready to run.
+    PRE_SETUP : int
+        Newly initialized problem or newly added model.
+    POST_CONFIGURE : int
+        Configure has been called.
```

Comment:
This status is never used, and I am confused about just what it means from a problem perspective. @naylor-b

Reply:
Yeah I agree there's a little confusion here, and I commented earlier that these constants should not be attributes of Problem.  I think for clarity we may want to differentiate between Problem setup and System setup, or possibly instead of system setup we should have a constant for POST_SETUP_PROCS instead.  That would be enough info to tell us when we're done with all of the system setup stuff (which all happens in setup_procs).

Reply:
So I've put the setup statuses in a constants file. Also, `POST_CONFIGURE` is used in group, I just wasn't referencing it as such because I didn't have `constants.py` at the time